### PR TITLE
ADD increase_tercount for TER

### PR DESCRIPTION
**Pull request by hainm**
What do you think? This would help  pdb4amber writing correct CONECT record. 

From:
```
ATOM      1  N   CYX L   1      57.464  29.769  15.871  1.00 25.39           N
ATOM      2  SG  CYX L   1      56.982  27.807  18.150  1.00 14.20           S
TER       3      CYX L   1
ATOM      4  N   CYX H   2      36.233  17.035  12.739  1.00 10.49           N
ATOM      5  SG  CYX H   2      36.833  15.443  15.640  1.00 15.60           S
```

to 
```
ATOM      1  N   CYX L   1      57.464  29.769  15.871  1.00 25.39           N  
ATOM      2  SG  CYX L   1      56.982  27.807  18.150  1.00 14.20           S  
TER       2      CYX L   1  # <= CHANGE HERE 
ATOM      3  N   CYX H   2      36.233  17.035  12.739  1.00 10.49           N  
ATOM      4  SG  CYX H   2      36.833  15.443  15.640  1.00 15.60           S  
END                                                                             
```

### CODE REVIEW

**Comment by hainm** [comment]
run jenkins

File: parmed/formats/pdb.py [hunk 1]
```diff
@@ -736,6 +737,9 @@ def write(struct, dest, renumber=True, coordinates=None, altlocs='all',
         standard_resnames : bool, optional, default False
             If True, common aliases for various amino and nucleic acid residues
             will be converted into the PDB-standard values.
+        increase_tercount : bool, optional, default True
+            If True, the TER atom number field increased by one compared to
+            atom card preceding it.
```
Comment (swails) [inline parmed/formats/pdb.py]: Add a comment here that `True` conforms to PDB standard.  That's the reason this was the default behavior before.

**Comment by hainm** [comment]
Hey man, what’s the status of this pr?

**Comment by swails** [comment]
LGTM -- I'm just a slow merger these days

**Comment by hainm** [comment]
thanks man.

**Pull request merged**